src/api_cmds.go

```diff
@@ -139,7 +139,7 @@ type commandContext struct {
 // creates commandContext, changes the process' working directory to
 // file's directory and ensures that a build has been made. It is
 // meant to be used by user-facing commands.
-func prepareCommandContext(file string) (commandContext, error) {
+func prepareCommandContext(file string, ensure_build bool) (commandContext, error) {
```

Comment:
Go style uses mixed caps casing for identifiers, see [here](https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps).

Also, the documentation should be updated accordingly, since this is no longer always true:

> and ensures that a build has been made.